### PR TITLE
Skip new installations when doing a hibernation check

### PR DESCRIPTION
**Pull request by gabrieljackson**
The creation timestamp of an installation is now checked when
performing hibernations to efficiently skip installations that are
newer than the provided day inactivity value.

Fixes https://mattermost.atlassian.net/browse/MM-36810

### CODE REVIEW

File: cmd/fleet-controller/hibernate.go [hunk 1]
```diff
@@ -93,6 +93,8 @@ var hibernate = &cobra.Command{
 		logger.Infof("Calculating hibernate actions on %d stable installations", len(installations))
 		var errorSkipCount, maxUserSkipCount int
 		var installationsToHibernate []*cmodel.InstallationDTO
+		creationTimestampCutoff := (time.Now().UnixNano() / int64(time.Millisecond)) - (int64(days) * 24 * int64(time.Millisecond))
```
Comment (Szymongib) [inline cmd/fleet-controller/hibernate.go:96]: I believe this calculation is a little off.
Given `days = 1` the `int64(days) * 24 * int64(time.Millisecond)` will result in `24000000` whereas one day is `86400000` milliseconds.
```suggestion
		creationTimestampCutoff := (time.Now().UnixNano() / int64(time.Millisecond)) - (int64(days) * 24 * int64(time.Hour / time.Millisecond))
```

Comment (gabrieljackson) [reply cmd/fleet-controller/hibernate.go]: Yep, you are totally right. I had tested this and forgot to put it back. Thanks for catching it.

**Review by Szymongib — APPROVED** [review]
LGTM!

**Review by angeloskyratzakos — APPROVED** [review]
LGTM nice addition Gabe!

**Pull request merged**